fed/config.py

```diff
@@ -48,23 +48,25 @@ def cross_silo_comm_config_dict(self) -> Dict:
 _job_config = None
 
 
-def get_cluster_config():
+def get_cluster_config(job_name: str = None):
```

Comment:
Hum， so this method name should be changed to get_job_config or get_config?

Reply:
Both "cluster_config" and "job_config" are part of the JOB, see #156. So I think it's fine to retrieve a "cluster config" by "job name" though it's definitely hard to understand 🤣

I think in the near future, we can merge these two configs.

Reply:
OK. Then let us leave a TODO comment on that target?

Reply:
Sure